corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/util/util.js

```diff
@@ -0,0 +1,24 @@
+function Util() {
+}
+
+Util.getQueryParams = function (qs) {
+    qs = qs.split("+").join(" ");
+
+    var params = [], tokens,
+        re = /[?&]?([^=]+)=([^&]*)/g;
+
+    while (tokens = re.exec(qs)) {
```

Comment:
Expected a conditional expression and instead saw an assignment. (no-cond-assign)